pkg/api/handlers/libpod/play.go

```diff
@@ -70,14 +70,14 @@ func PlayKube(w http.ResponseWriter, r *http.Request) {
 		password = authConf.Password
 	}
 
-	logDriver := query.LogDriver
-	if logDriver == "" {
+	var logDriver string
```

Comment:
It should still be initialized with `query.LogDriver`.  Currently, `query.LogDriver` is ignored.